docs/guides/cli-completions.md

```diff
@@ -0,0 +1,39 @@
+# CLI Completions
+`lockbook` ships with a `_lockbook` tab completions file.
+In general, your shell scans `$FPATH` for completions.
+
+## Debugging CLI Completions
+If `lockbook` completions are not working automatically for you, check out [homebrew's](https://docs.brew.sh/Shell-Completion) guide.
```

Comment:
this advice is only relevant for macos && (zsh || bash), possibly the structure of the document should reflect that